Adyen/Components/Base/AddressInfo.swift

```diff
@@ -9,6 +9,26 @@ import Foundation
 /// The model for address data.
 public struct AddressInfo: Equatable, Encodable {
 
+    internal let invalidCountry = "ZZ"
+    internal let invalidValue = "N.A."
```

Comment:
maybe these could be static constants, since they relate to the whole type.

Reply:
👍